Service/Transliterator.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace LePhare\Import\Service;
```

Comment:
It's not a service and it does not make sense outside of a Symfony bundle.

```suggestion
namespace LePhare\Import;
```